BrownianMotion/Continuity/IsKolmogorovProcess.lean

```diff
@@ -181,6 +181,20 @@ lemma IsKolmogorovProcess.edist_eq_zero (hX : IsKolmogorovProcess X P p q M)
   _ ≤ M * edist s t ^ q := hX.kolmogorovCondition s t
   _ = 0 := by simp [h, hq]
 
+lemma IsKolmogorovProcess.M_eq_zero (hX : IsKolmogorovProcess X P p q 0)
```

Comment:
The name `edist_eq_zero` for the other lemma was supposed to describe the result (omitting the a.e.), not the hypothesis. Here the result is the same, so here is a suggestion:
```suggestion
lemma IsKolmogorovProcess.edist_eq_zero_of_M_eq_zero (hX : IsKolmogorovProcess X P p q 0)
```